.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "4"
-  - "6"
   - "8"
```

Comment:
drop 8?

Reply:
Maybe next time when we add 14 :-D. Decided to keep three supported versions.